rutil/Data.cxx

```diff
@@ -352,6 +352,16 @@ Data::Data(const string& str)
    initFromString(str.c_str(), (Data::size_type)str.size());
 }
 
+#if RESIP_CPP_STANDARD >= 201703L
+/**
+  Creates a data with the contents of the string_view.
+*/
+ Data::Data(const std::string_view sv)
```

Comment:
Should probably be a reference, ie: (const std::string_view& sv)

Reply:
Based on my understanding and research, it’s now generally preferred to use std::string_view for parameters. It allows the compiler to optimize to resulting in better efficiency.